lib/config.js

```diff
@@ -1,7 +1,8 @@
 /*!
- * Copyright (c) 2016-2017 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2016-2018 Digital Bazaar, Inc. All rights reserved.
  */
 const bedrock = require('bedrock');
+const path = require('path');
```

Comment:
I think we've been trying to keep requires in alphabetical order.  And since we're here, `config` can be destructured.  Maybe `bedrock` is not even required.